File: design/html-and-assets/project/tokens/YarngoTokens.swift
// YarngoTokens.swift — Yarngo Design Tokens v1.0
import SwiftUI

public enum Yarngo {
    public enum Color {
        // Brand
        public static let orange     = SwiftUI.Color(hex: 0xFF8A1F)
        public static let orange600  = SwiftUI.Color(hex: 0xE6740F)
        public static let orange700  = SwiftUI.Color(hex: 0xBF5C08)
        public static let ink        = SwiftUI.Color(hex: 0x171717)
        public static let warmWhite  = SwiftUI.Color(hex: 0xFFF9F2)
        public static let leaf       = SwiftUI.Color(hex: 0x287A57)
        public static let signal     = SwiftUI.Color(hex: 0x596DF8)
        // Semantic — light
        public static let bg         = SwiftUI.Color(hex: 0xFFF9F2)
        public static let surface    = SwiftUI.Color(hex: 0xFFFFFF)
        public static let border     = SwiftUI.Color(hex: 0xEBE4D9)
        public static let text       = SwiftUI.Color(hex: 0x171717)
        public static let textMuted  = SwiftUI.Color(hex: 0x5F594F)
        public static let onAccent   = SwiftUI.Color(hex: 0x171717) // Ink on orange, never white
        public static let success    = SwiftUI.Color(hex: 0x287A57)
        public static let warning    = SwiftUI.Color(hex: 0xC97A00)
        public static let danger     = SwiftUI.Color(hex: 0xC7362B)
        // Semantic — dark
        public enum Dark {
            public static let bg        = SwiftUI.Color(hex: 0x121110)
            public static let surface   = SwiftUI.Color(hex: 0x1F1C19)
            public static let border    = SwiftUI.Color(hex: 0x35302A)
            public static let text      = SwiftUI.Color(hex: 0xFFF9F2)
            public static let accent    = SwiftUI.Color(hex: 0xFF9C3D)
        }
    }
    public enum Space {
        public static let x1: CGFloat = 4,  x2: CGFloat = 8,  x3: CGFloat = 12
        public static let x4: CGFloat = 16, x5: CGFloat = 20, x6: CGFloat = 24
        public static let x8: CGFloat = 32, x10: CGFloat = 40, x12: CGFloat = 48
    }
    public enum Radius {
        public static let xs: CGFloat = 4,  sm: CGFloat = 8,  md: CGFloat = 12
        public static let lg: CGFloat = 16, xl: CGFloat = 24, xxl: CGFloat = 32
        public static let full: CGFloat = 999
    }
    public enum Duration {
        public static let instant = 0.12, fast = 0.18, base = 0.24, slow = 0.32, breathe = 1.6
    }
    public enum Font {
        public static func display(_ size: CGFloat, _ weight: SwiftUI.Font.Weight = .semibold) -> SwiftUI.Font {
            .custom("Sora", size: size).weight(weight)
        }
        public static func text(_ size: CGFloat, _ weight: SwiftUI.Font.Weight = .regular) -> SwiftUI.Font {
            .custom("NotoSans", size: size).weight(weight)
        }
    }
    public static let minTouchTarget: CGFloat = 44
}

extension SwiftUI.Color {
    init(hex: UInt32) {
        self.init(.sRGB,
                  red:   Double((hex >> 16) & 0xFF) / 255,
                  green: Double((hex >> 8)  & 0xFF) / 255,
                  blue:  Double( hex        & 0xFF) / 255,
                  opacity: 1)
    }
}

File: design/html-and-assets/project/tokens/YarngoTokens.kt
// YarngoTokens.kt — Yarngo Design Tokens v1.0
package com.yarngo.design

import androidx.compose.ui.graphics.Color
import androidx.compose.ui.unit.dp
import androidx.compose.ui.unit.sp

object YarngoColor {
    // Brand
    val Orange    = Color(0xFFFF8A1F)
    val Orange600 = Color(0xFFE6740F)
    val Orange700 = Color(0xFFBF5C08)
    val Ink       = Color(0xFF171717)
    val WarmWhite = Color(0xFFFFF9F2)
    val Leaf      = Color(0xFF287A57)
    val Signal    = Color(0xFF596DF8)
    // Semantic — light
    val Bg        = Color(0xFFFFF9F2)
    val Surface   = Color(0xFFFFFFFF)
    val Border    = Color(0xFFEBE4D9)
    val Text      = Color(0xFF171717)
    val TextMuted = Color(0xFF5F594F)
    val OnAccent  = Color(0xFF171717) // Ink on orange, never white
    val Success   = Color(0xFF287A57)
    val Warning   = Color(0xFFC97A00)
    val Danger    = Color(0xFFC7362B)
    // Semantic — dark
    object Dark {
        val Bg      = Color(0xFF121110)
        val Surface = Color(0xFF1F1C19)
        val Border  = Color(0xFF35302A)
        val Text    = Color(0xFFFFF9F2)
        val Accent  = Color(0xFFFF9C3D)
    }
}

object YarngoSpace {
    val x1 = 4.dp; val x2 = 8.dp;  val x3 = 12.dp; val x4 = 16.dp
    val x5 = 20.dp; val x6 = 24.dp; val x8 = 32.dp; val x10 = 40.dp; val x12 = 48.dp
}

object YarngoRadius {
    val xs = 4.dp; val sm = 8.dp; val md = 12.dp
    val lg = 16.dp; val xl = 24.dp; val xxl = 32.dp; val full = 999.dp
}

object YarngoDuration {
    const val Instant = 120; const val Fast = 180; const val Base = 240
    const val Slow = 320; const val Breathe = 1600
}

object YarngoType {
    val Display1 = 64.sp; val Display2 = 48.sp; val H1 = 36.sp; val H2 = 28.sp
    val H3 = 22.sp; val Body = 16.sp; val BodySm = 14.sp; val Label = 12.sp
}

val YARNGO_MIN_TOUCH_TARGET = 44.dp
